### ISSUE TITLE
Inconsistent messages for attachments

### ISSUE THREAD

**Issue by pavelevap** [versionpress/versionpress#878]
![different_messages](https://cloud.githubusercontent.com/assets/3008007/14332019/10dc8ff2-fc48-11e5-91db-4165dc2ec4aa.jpg)

1) Updated `post_content`: "Edited attachment" (2x).
2) Updated `_wp_attachment_image_alt`: "Updated attachment", "Edited attachment" and in the end "Created postmeta for post" (and not attachment).
3) Updated `post_excerpt`: "Updated attachment" and "Edited attachment".

And related note: When user works in media modal window there are several fields (Title, Caption, Alt Text, Description) and every change is different commit (probably because of Ajax calls). It would be nice to commit it only once in this modal, but I am not sure if it is even possible somehow :-)

**Comment by octopuss** [comment]
@pavelevap If I understand you correctly, we should always set commit message to "Edited" or "Updated" to be consistent. Am I right ?

Unfortunately we are not able to simple merge ajax requests into one. What do you think @JanVoracek ?

**Comment by pavelevap** [comment]
Exactly, same actions should have same messages. Also we should not talk about post (Created postmeta for "post") when editing attachment.

Media modals are Backbone templates so I thought we could use event handlers [somehow](http://wordpress.stackexchange.com/questions/124005/is-it-possible-to-trigger-some-javascript-when-media-popup-is-opened) to check if user is still active in this modal window, postpone Ajax commits and make only one commit when user closes window (or moves to another image). But I am not sure about possibility of technical solution, it is only an idea :-) I edited several images I received many commits, because everytime I click to another field, Ajax call is processed.

**Comment by borekb** [comment]
Related: 'edit' -> 'update' action verb #1120

**Comment by borekb** [comment]
Will be resolved with Rich action messages (markup) #1124
